### PR TITLE
ELEMENTS-737: skip fwd'ing filter host props on clear

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: ui/nuxeo-filter.js [hunk 1]
```diff
@@ -269,6 +269,15 @@ import { FiltersBehavior } from './nuxeo-filters-behavior.js';
 
     _forwardHostPropV2(prop, value) {
       if (this._instance) {
+        const pendingUpdate = this._isPropertyPending('document') || this._isPropertyPending('user');
+        if (pendingUpdate) {
+          // if we have a pending update
+          const toClear = !this.check(); // let's check if instance will be removed
```
Comment (Gabez0r) [inline ui/nuxeo-filter.js]: This seems to be the approach to go. What about the idea of caching the `this.check()` result?

Comment (nelsonsilva) [reply ui/nuxeo-filter.js]: as all caching... tricky invalidation... check only happens when `document` or `user` are pending so not for all host properties thus not that costly 🤞

Comment (Gabez0r) [reply ui/nuxeo-filter.js]: 🤞

**Comment by nelsonsilva** [comment]
https://qa.nuxeo.org/jenkins/job/Other/job/web-ui-pipeline-nsilva/126 🔵

**Pull request merged**